include/luabind/user_data.hpp

```diff
@@ -59,6 +59,10 @@ class user_data {
         lua_rawset(L, table_idx);
     }
 
+    static void get_custom_table(lua_State* L, int idx) {
```

Comment:
Please also add `set_custom_table()` function, which pops value from stack and sets it as a custom table for value at idx.

Reply:
done